bcompiler/compile.py

```diff
@@ -60,12 +60,15 @@ def parse_source_cells(source_file: str, datamap_source_file: str) -> \
     datamap_obj = Datamap()
     datamap_obj.cell_map_from_csv(datamap_source_file)
     for item in datamap_obj.cell_map:
+        if item.template_sheet == "":
+            continue
         if item.template_sheet is not None and item.cell_reference is not None:
             try:
                 ws = wb[item.template_sheet]
             except KeyError as e:
                 logger.critical(f"{e}.{source_file} is not a BICC template. Not processing. Remove it!")
                 sys.exit()
```

Comment:
You should maybe amend this to remove `sys.exit()` (which quits the program) and leave the CRITICAL log message  on line 69 which will flag a potentially catestrophic failure (trying to process a sheet that isn't there) but try to continue processing anyway.

Reply:
from what I can tell the programme seems to be working fine without the extra change you suggest above. So up to you whether this change is necessary or not.